network/network_test.go

```diff
@@ -230,6 +232,8 @@ func TestNetwork(t *testing.T) {
 	})
 
 	t.Run("all nodes have at least one connection to the bootstrap node B", func(t *testing.T) {
+		fmt.Println(t.Name())
```

Comment:
You can use `t.Log(t.Name())` instead fmt.Println.

Reply:
Done with thanks